app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -7,14 +7,15 @@ def host_overview_buttons(host)
     end
 
     def multiple_actions
-      super + [ [_('Schedule Remote Job'), new_job_invocation_path, false] ]
+      super + [ [_('Schedule Remote Job'), new_job_invocation_path, false] ] +
+              user_job_actions.map { |a| a.push(false) }
```

Comment:
Could we avoid in-place modification? Something like the following might give more clarity what's going on on the first sight
```ruby
user_job_actions.map { |(name, path)| [name, path, false] }
```